src/admin/components/website/WebsiteEditors.tsx

```diff
@@ -0,0 +1,161 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+import * as React from 'react';
+import {connect} from 'react-redux';
+
+class WebsiteEditors extends React.Component <any, any> {
+	constructor (props) {
+		super(props);
+
+		this.state = {
+			editMode: false,
+		};
+
+		this.onToggleEdit = this.onToggleEdit.bind(this);
+		this.onDelete = this.onDelete.bind(this);
+		this.onChange = this.onChange.bind(this);
+		this.checkHost = this.checkHost.bind(this);
+		this.getUsers = this.getUsers.bind(this);
+		this.isEditing = this.isEditing.bind(this);
+	}
+
+	onToggleEdit () {
+		return this.setState({editMode: !this.state.editMode});
+	}
+
+	onDelete (index) {
+		const chiefEditors = this.props.chiefEditors.asMutable();
+		chiefEditors.splice(index, 1);
+		return this.props.onSubmit({chiefEditors});
+	}
+
+	onChange (e) {
+		const userID = e.target.value;
+		if (userID) {
+			let newUser = null;
+			this.props.users.forEach((user) => {
+				if (user.ID === userID) {
+					newUser = user.without(['ID', 'date']);
+				}
+			});
+			const chiefEditors = (typeof this.props.chiefEditors.asMutable !== 'undefined') ?
+									this.props.chiefEditors.asMutable() : this.props.chiefEditors;
+			chiefEditors.push(newUser);
+			e.target.value = '';
```

Comment:
We do not use state in this component and on change we add user to the list. Because modal window does not close on it - we have to drop the selected value, because it would not be in a select list any more